website/cue/reference/components/sinks/loki.cue

```diff
@@ -105,26 +105,43 @@ components: sinks: loki: {
 		label_expansion: {
 			title: "Label Expansion"
 			body: """
-				The `labels` option can be passed keys suffixed with `*` to
-				allow for setting multiple keys based on the contents of an
-				object. For example, with an object:
+				The `labels` option can be passed keys with `*` or prefixes ending with `*` to
+				allow for setting multiple keys based on the contents of an object. Static keys
+				override dynamically defined keys. For example, with an object:
```

Comment:
🤔 I didn't even think about this, it _may_ be worth having this in the configuration description itself - as well as clarifying what happens if two dynamic labels have the same key. I think the last "configured" key would win?

Reply:
I've not tested with dynamic duplicates, but assume that it would/should be last-wins (at least I'd expect that with map structures).

Within the labels there is a `Vec<(String, String)>` involved which made the situation unclear and would have allowed  for duplicates between static/dynamic labels during processing. The implementation prefers the static labels over the dynamic ones.